boot-mongodb-elasticsearch/src/main/java/com/example/mongoes/web/controller/RestaurantController.java

```diff
@@ -26,79 +30,79 @@
 @RestController
 @Timed
 @RequestMapping("/api/restaurant")
-public class RestaurantController {
+@Validated
+class RestaurantController {
```

Comment:
@coderabbitai, can you please help me with writing openapi swagger spec for all endpoints in this controller